src/jdk.internal.vm.ci/share/classes/jdk/vm/ci/hotspot/HotSpotJVMCIRuntime.java

```diff
@@ -221,35 +221,60 @@ static String[] exceptionToString(Throwable o, boolean toString, boolean stackTr
+     * Javadoc is used instead to document these options.
+     */
+    private static final String[] NO_HELP = null;
+
     /**
      * A list of all supported JVMCI options.
      */
     public enum Option {
         // @formatter:off
-        Compiler(String.class, null, "Selects the system compiler. This must match the getCompilerName() value returned " +
-                "by a jdk.vm.ci.runtime.JVMCICompilerFactory provider. " +
-                "An empty string or the value \"null\" selects a compiler " +
-                "that will raise an exception upon receiving a compilation request."),
-        // Note: The following one is not used (see InitTimer.ENABLED). It is added here
-        // so that -XX:+JVMCIPrintProperties shows the option.
-        InitTimer(Boolean.class, false, "Specifies if initialization timing is enabled."),
-        CodeSerializationTypeInfo(Boolean.class, false, "Prepend the size and label of each element to the stream when " +
-                "serializing HotSpotCompiledCode to verify both ends of the protocol agree on the format. " +
-                "Defaults to true in non-product builds."),
-        DumpSerializedCode(String.class, null, "Dump serialized code during code installation for code whose simple " +
-                "name (a stub) or fully qualified name (an nmethod) contains this option's value as a substring."),
-        ForceTranslateFailure(String.class, null, "Forces HotSpotJVMCIRuntime.translate to throw an exception in the context " +
-                "of the peer runtime. The value is a filter that can restrict the forced failure to matching translated " +
-                "objects. See HotSpotJVMCIRuntime.postTranslation for more details. This option exists solely to test " +
-                "correct handling of translation failures."),
-        PrintConfig(Boolean.class, false, "Prints VM configuration available via JVMCI."),
-        AuditHandles(Boolean.class, false, "Record stack trace along with scoped foreign object reference wrappers " +
-                "to debug issue with a wrapper being used after its scope has closed."),
-        TraceMethodDataFilter(String.class, null,
-                "Enables tracing of profiling info when read by JVMCI.",
-                "Empty value: trace all methods",
-                        "Non-empty value: trace methods whose fully qualified name contains the value."),
-        UseProfilingInformation(Boolean.class, true, "");
+        Compiler(String.class, null,
+                "Selects the system compiler. This must match the getCompilerName() value",
+                "returned by a jdk.vm.ci.runtime.JVMCICompilerFactory provider. ",
+                "An empty string or the value \"null\" selects a compiler ",
+                "that raises an exception upon receiving a compilation request."),
+
+        PrintConfig(Boolean.class, false, "Prints VM values (e.g. flags, constants, field offsets etc) exposed to JVMCI."),
```

Comment:
```suggestion
        PrintConfig(Boolean.class, false, "Prints VM values (such as options, constants, and field offsets) exposed to JVMCI."),
```
I'm trying to be consistent with the use of "option" (instead of "flag").